megamek/src/megamek/common/Aero.java

```diff
@@ -4385,4 +4392,82 @@ public void updateSensorOptions() {
             }
         }
     }
+    
+    /**
+     * @return Returns the autoEject.
+     */
+    public boolean isAutoEject() {
+        if (game.getOptions().booleanOption(OptionsConstants.ADVANCED_STRATOPS_QUIRKS)) {
+            return !hasQuirk(OptionsConstants.QUIRK_NEG_NO_EJECT);
+        }
+        return true;
```

Comment:
The player is allowed to toggle autoeject, but the player's selection is ignored.

Reply:
Fixed